lib/process-services-cloud/src/lib/process-services-cloud.module.ts

```diff
@@ -87,9 +87,12 @@ export class ProcessServicesCloudModule {
         };
     }
 
-    static forChild(): ModuleWithProviders<ProcessServicesCloudModule> {
+    static forChild(preferenceServiceInstance?: PreferenceCloudServiceInterface): ModuleWithProviders<ProcessServicesCloudModule> {
```

Comment:
We need to be able to provide preferences `UserPreferenceCloudService` service.

This is done in the same way as for root: https://github.com/Alfresco/alfresco-ng2-components/pull/7568/files#diff-d7355a313d50b1e32db0269c4a1e95720dde82f661ce21fdb6b84a5e29133c39R69-R89

Reply:
I removed it in next PR in part [3/3], so it stays as it was

https://github.com/Alfresco/alfresco-ng2-components/pull/7569/files#diff-d7355a313d50b1e32db0269c4a1e95720dde82f661ce21fdb6b84a5e29133c39R97-R99